docs/fuse.md

```diff
@@ -26,6 +28,12 @@ go get github.com/jbenet/go-fuse-version/fuse-version
 
 If you run into any problems installing FUSE or mounting IPFS, hop on IRC and speak with us, or if you figure something new out, please add to this document!
 
+You may also have to add your username to the `operator` group:
```

Comment:
could we describe more why this has to happen? or if there is another way to do it? some users will not want to modify user groups.